phase1/main.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python
+#
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import urllib
+
+from google.appengine.ext import ndb
+from google.appengine.api import users
+from google.appengine.ext.webapp import template
+import webapp2
+
+
+# Global variables
+API_KEY = "your-api-key"
```

Comment:
I think we should use the usable auth. You can use the App Engine service account on App Engine production, and you can use a generic service account on devserver (it only works with gcloud preview app IIRC)